cvm/quantization/sym_pass.py

```diff
@@ -735,24 +735,79 @@ def _reduce_graph(sym, params, graph, inputs_ext):
     return node, params
 
 def fuse_transpose(symbol, params, logger=logging):
-    # node                           node            node
-    #  |                 reduce       |
-    # transpose1       =========>  transpose    or
-    #  |
-    # transpose2
     def _fuse_transpose(sym, params, graph, inputs_ext):
-        if sym.attr('op_name') == 'transpose':
-            axes, consec = eval(sym.attr('axes')), False
-            childs = sym_iter(sym.get_children())
+        name, op = sym.attr('name'), sym.attr('op_name')
+        childs = sym_iter(sym.get_children())
```

Comment:
```suggestion
        childs = sym_iter(sym.get_children())
        attr = sym.list_attr()
```
You can get attributes by list_attr function, and then get attr with the dict.

the `get_attr` function in *sym_utils.py* has wrapped the requirements of
getting the **evaluate**(that is `eval` function) value by key in dict 
and return a default value if needed or throw an exception.
- get_attr('axis') will throw exception if not set `axis`;
- get_attr('axis', None) will return None if not set `axis`;

It will be user-unfriendly since `sym.attr` function returns None even if the symbol doesn't set the attribute instead of throwing an exception to warning user.

Need to change such as line 750, line 764 and I will note this situation for following code.